source/guide_filebrowser.rst

```diff
@@ -0,0 +1,129 @@
+.. highlight:: console
+
+.. author:: on4r <https://github.com/on4r>
+
+.. categorize your guide! refer to the current list of tags: https://lab.uberspace.de/tags
+.. tag:: web
+.. tag:: file-storage
+.. tag:: self-hosting
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/filebrowser.png
+      :align: center
+
+##########
+File Browser
+##########
+
+.. tag_list::
+
+filebrowser_ provides a file managing interface within a specified directory and it can be used to upload, delete, preview, rename and edit your files.
+It allows the creation of multiple users and each user can have its own directory. It can be used as a standalone app or as a middleware.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`web backends <web-backends>`
+  * :manual:`supervisord <daemons-supervisord>`
+
+License
+=======
+
+filebrowser is licensed under the `Apache License 2.0`_
+
+Installation
+============
+
+Download and extract the `latest release`_
+-------------------
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cd ~/bin
+ [isabell@stardust bin]$ wget <url_of_linux_amd64_filebrowser>
+ [isabell@stardust bin]$ tar -xzf linux-amd64-filebrowser.tar.gz
+
+Initialize a new database
+-------------------------
+
+.. code-block:: console
+
+ [isabell@stardust bin]$ cd ~
+ [isabell@stardust ~]$ mkdir filebrowser && cd filebrowser
+ [isabell@stardust filebrowser]$ filebrowser config init
+
+
+Configuration
+=============
+
+.. note:: You have to be in the same directory as the *filebrowser.db* file which you created before, in order to configurate it or else it will create a new one.
+.. note:: Check your running web backends (``uberspace web backend list``) and choose a free port between 1024 and 65535.
+
+.. code-block:: console
+
+ [isabell@stardust filebrowser]$ filebrowser config set --port 1337
+ [isabell@stardust filebrowser]$ filebrowser config set --address 0.0.0.0
+ [isabell@stardust filebrowser]$ filebrowser config set --baseurl /my_files
+ [isabell@stardust filebrowser]$ filebrowser config set --root /home/<username>/html/my_files
```

Comment:
- We don't need to change the port, the default port `8080` is also working.
```
--port string                        port to listen on (default "8080")
```
- You may add a short info at some point:
 note: changes to the config via CLI can not be done while the server is running as the database access is blocked by the instance.
- Why do you use a subfolder for the `baseurl`? As per style guide you need to keep it simple.
 You even remove the that prefix with the web backend, so it actually makes no sense to me.

This is how I did it:

```bash
cd ~/bin
VERSION=2.22.4
curl --location https://github.com/filebrowser/filebrowser/releases/download/v$VERSION/linux-amd64-filebrowser.tar.gz | tar -xzf - filebrowser
mkdir ~/filebrowser ~/my_files && cd ~/filebrowser
filebrowser config init
filebrowser config set --address 0.0.0.0
filebrowser config set --root $HOME/my_files
filebrowser users add '<username>' '<password>' --perm.admin
uberspace web backend set / --http --port 8080
```

Reply:
- `8080` is only working if there are no other backends running which use this port. I think it's good practice to first check what's already running and used and then choose a port. 
- in my personal setup i have many different services and applications running at different urls, thats why i use `baseurl`. About the prefix thou... yeah, it doesn't make any sense. I guess we can keep it simple here and go with your suggestion.

Reply:
Every uberspace (V7) user has its separate and conflict free port network. As I have checked it looks like the port 8080 is free to use at a fresh uberspace instance, so I don't see a problem. - You can't even reach another users network services without opening extra firewall ports with the uberspace command line tool. - Only these open ports (we don't need in this case) will be proxied to the host network and then are open to the galaxy wide internet. 😉
Have a look at https://manual.uberspace.de/background-network/

The lab guides and the uberspaces are ment to be used as single app containers, so for each service you are running you should run an extra uberspace (containers). So port conflicts with other of you own services should not be a thing in context of these lab guides.

Reply:
I wasn't talking about conflicts with other users ports but with my own running applications. But thanks for the reminder. 
I read somewhere in your docs about putting each service onto its own uberspace account. While i don't think it's a practicable approach for the default user, i see why we should stay in context of the lab guides.